packages/evm/src/precompiles/index.ts

```diff
@@ -6,9 +6,9 @@ import { precompile02 } from './02-sha256.js'
 import { precompile03 } from './03-ripemd160.js'
 import { precompile04 } from './04-identity.js'
 import { precompile05 } from './05-modexp.js'
-import { precompile06 } from './06-ecadd.js'
-import { precompile07 } from './07-ecmul.js'
-import { precompile08 } from './08-ecpairing.js'
+import { precompile06 } from './06-bn254-add.js'
+import { precompile07 } from './07-bn254-mul.js'
+import { precompile08 } from './08-bn254-pairing.js'
```

Comment:
I am confused now. The EIPs https://eips.ethereum.org/EIPS/eip-196 https://eips.ethereum.org/EIPS/eip-197 mention that these are "alt_bn128"? It seems that `bn254` is an alias for this? :thinking:

Reply:
Yes, these are the same things. It was called bn128 when it had 128-bit security, but then researchers decreased security bits. So it's now bn254, because of 254-bit field

Reply:
Yes, I followed Pauls advice here, also googling a bit and stumbling upon articles like https://hackmd.io/@jpw/bn254 for some confirmation that this is the "official" respectively "cryptographically correct" name now.

This naming confusion is a thing though, to add on this: there is even the name "bn256" (!!) around, this seems to be wrong and there is some explanation what happened to these two bits I forgot again. 😂 Geth is using this (wrong) name though, I just discovered yesterday (which should not bother us).

But let's stop here. This is really not something where we can clean up the mess. So: bn254 seems very much to be the best choice here.